install/install.sh

```diff
@@ -17,6 +17,12 @@ set -e
 # Check if the script is running as root
 [[ $EUID != 0 ]] && echo "Script must run as root."  && exit 1
 
+echo "Check if network and remote are available."
+curl -fsSL --silent $REMOTE/install/install.sh &> /dev/null || (
```

Comment:
I think we should only redirect stdout, stderr would be useful here?

Reply:
done